Sources/OpenAPIRuntime/Errors/RuntimeError.swift

```diff
@@ -73,6 +74,8 @@ internal enum RuntimeError: Error, CustomStringConvertible, LocalizedError, Pret
             return "Invalid expected content type: '\(string)'"
         case .invalidHeaderFieldName(let name):
             return "Invalid header field name: '\(name)'"
+        case .invalidBase64String(let string):
+            return "Invalid base64-encoded string: '\(string)'"
```

Comment:
Let's cap how many bytes we'd actually print.
```suggestion
            return "Invalid base64-encoded string (first 128 bytes): '\(string.prefix(128))'"
```